docs/examples/stocks-portfolio.md

```diff
@@ -0,0 +1,404 @@
+   "maxSupply":"None",
+   "openInterest":"None",
+   "marketCap":40567812096,
+   "volumeAllCurrencies":"None",
+   "strikePrice":"None",
+   "averageVolume":2200304,
+   "priceToSalesTrailing12Months":57.123283,
+   "dayLow":147.61,
+   "ask":152,
+   "ytdReturn":"None",
+   "askSize":800,
+   "volume":1362006,
+   "fiftyTwoWeekHigh":174.94,
+   "forwardPE":"None",
+   "fromCurrency":"None",
+   "fiveYearAvgDividendYield":"None",
+   "fiftyTwoWeekLow":65.11,
+   "bid":148,
+   "tradeable":false,
+   "dividendYield":"None",
+   "bidSize":1800,
+   "dayHigh":156.92,
+   "exchange":"NYQ",
+   "shortName":"Unity Software Inc.",
+   "longName":"Unity Software Inc.",
+   "exchangeTimezoneName":"America/New_York",
+   "exchangeTimezoneShortName":"EST",
+   "isEsgPopulated":false,
+   "gmtOffSetMilliseconds":"-18000000",
+   "quoteType":"EQUITY",
+   "symbol":"U",
+   "messageBoardId":"finmb_241908542",
+   "market":"us_market",
+   "annualHoldingsTurnover":"None",
+   "enterpriseToRevenue":54.824,
+   "beta3Year":"None",
+   "profitMargins":-0.35116002,
+   "enterpriseToEbitda":-213.991,
+   "52WeekChange":1.1919534,
+   "morningStarRiskRating":"None",
+   "forwardEps":"None",
+   "revenueQuarterlyGrowth":"None",
+   "sharesOutstanding":270776992,
+   "fundInceptionDate":"None",
+   "annualReportExpenseRatio":"None",
+   "bookValue":"None",
+   "sharesShort":6390454,
+   "sharesPercentSharesOut":0.023599999,
+   "fundFamily":"None",
+   "lastFiscalYearEnd":1577750400,
+   "heldPercentInstitutions":0.61965,
+   "netIncomeToCommon":-317163008,
+   "trailingEps":"None",
+   "lastDividendValue":"None",
+   "SandP52WeekChange":0.14322305,
+   "priceToBook":"None",
+   "heldPercentInsiders":0.16246,
+   "nextFiscalYearEnd":1640908800,
+   "mostRecentQuarter":1601424000,
+   "shortRatio":3.2,
+   "sharesShortPreviousMonthDate":1607990400,
+   "floatShares":120820703,
+   "enterpriseValue":38934634496,
+   "threeYearAverageReturn":"None",
+   "lastSplitDate":"None",
+   "lastSplitFactor":"None",
+   "legalType":"None",
+   "lastDividendDate":"None",
+   "morningStarOverallRating":"None",
+   "earningsQuarterlyGrowth":"None",
+   "dateShortInterest":1610668800,
+   "pegRatio":-15.31,
+   "lastCapGain":"None",
+   "shortPercentOfFloat":0.048600003,
+   "sharesShortPriorMonth":5086368,
+   "impliedSharesOutstanding":"None",
+   "category":"None",
+   "fiveYearAverageReturn":"None",
+   "regularMarketPrice":152.33,
+   "logo_url":"https://logo.clearbit.com/unity.com"
+}
+```
+
+We want the `['longName']`, `['regularMarketOpen']` and `['longBusinessSummary']`. Now we want the bot to send the data, so we will be using `await message.respond()`. Here is what it should look like:
+
+```python
+class StockSkill(Skill):
+
+    @match_regex(r"Stock: (.*)")
+    async def search_info_stock(self, message):
+        stock_name = message.regex.group(1)
+        stock = yf.Ticker(stock_name)
+        await message.respond(f"Name: {stock.info['longName']}")
```

Comment:
I'm happy to have it send more than one message, just as reference you could send a single message and make the string look like:

```python
await message.respond(f"Name: {stock.info['longName']}\nCurrent Price: {stock.info['regularMarketOpen']} {stock.info['currency']}\nAbout: {stock.info['longBusinessSummary']}")
```

Reply:
Thank you, I will be adding this